### ISSUE TITLE
The SQL editor should be in focus once I land on the page

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#11325]
- [ ] This element should be in focus state, with the cursor blinking. This makes it obvious I have to start writing SQL in the box.

Feedback was that its not clear what to do here. [There is another ticket to get an empty state](https://github.com/lightdash/lightdash/issues/11274) in, but I think the tiny change above would be really helpful!
